highlighting/lezer-grammars/level1.grammar

```diff
@@ -1,8 +1,9 @@
 @top Program { eol* (Command eol+)* Command? }
 Command {
-    Print | Ask | Echo | Turtle | ErrorInvalid
+    Print | Ask | Echo | Play | Turtle | ErrorInvalid
```

Comment:
This is great!  Now, we only need to edit `tokens.ts`. Since we can change the keywords at run time, we can't define the keywords directly in the grammar, therefore the need for a external specializer.

Now after compiling this parser, go to `tokens.ts` and modify the import for level 1 too add the play keyword. After this, go to the `keywordToToken` dictionary and add the keyword in the `extend` field. And at last, go to `cm-editor.ts` and edit the `hedyStyleTags` var to add the new keyword.

Reply:
Thank you!!

Reply:
Success!!

![image](https://github.com/hedyorg/hedy/assets/1003685/86e9edbb-ba5f-4553-9cbc-fb2251ffc5dc)

I found all of it except for the hedyStyleTags, so thanks a lot for that pointer!!